src/web/mjs/connectors/TenshiMoe.mjs

```diff
@@ -0,0 +1,154 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class TenshiMoe extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'tenshimoe';
+        super.label = 'Tenshi.moe';
+        this.tags = [ 'anime', 'english'];
+        this.url = 'https://tenshi.moe';
+        this.config = {
+            format: {
+                label: 'Language Settings',
+                description: 'Choose between Subbed or Dubbed anime. If your choice is not available the other is used as fallback',
+                input: 'select',
+                options: [
+                    { value: 'SUB', name: 'Subbed' },
+                    { value: 'DUB', name: 'Dubbed' }
+                ],
+                value: ''
+            },
+            resolution: {
+                label: 'Preferred Resolution',
+                description: 'Try to download video in the selected resolution.\nIf the resolution is not supported, depending on the mirror the download may fail, or a fallback resolution may be used!',
+                input: 'select',
+                options: [
+                    { value: '', name: 'Mirror\'s Default' },
+                    { value: '360', name: '360p' },
+                    { value: '480', name: '480p' },
+                    { value: '720', name: '720p' },
+                    { value: '1080', name: '1080p' }
+                ],
+                value: ''
+            }
+        };
+
+    }
+    async _initializeConnector() {
+        //bypass DdoSS Guard
+        const request = new Request(this.url, this.requestOptions);
+        const script = `
+        new Promise((resolve, reject) => {
+            setTimeout(() => {
+                    resolve();
+            },7000); //wait to let DG set its cookies
+        });
+        `;
+        await Engine.Request.fetchUI(request, script, 300000, true); //need image support, DG script loads a fake image
```

Comment:
Is `300000` by intention or a typo? 5 Minutes is quite the long timeout.

Reply:
typo